hw/ip/pwrmgr/doc/_index.md

```diff
@@ -238,26 +245,20 @@ The software is also able to enable recording during `Active` state if it choose
 
 ### Power-Up Handling
 
-The AST automatically takes flash out of power down state as part of the power manager's power up request.
-Once flash is powered up, the power manager keeps a running count of the flash initialization time.
-When this time has expired, an acknowledgment is forwarded to the flash controller.
-Once the boot ROM is allowed to execute, it is expected to initialize the flash prior to using it.
-This involves 4 steps:
+The AST automatically takes the flash storage out of power down state as part of the power manager's power up request.
+Once flash is powered up and ready, an indication is sent to the [flash_ctrl]({{< relref "hw/top_earlgrey/ip/ast/doc" >}}).
 
-*   Poll flash control register to ensure flash has powered up (this is the same signal mentioned above).
-*   Obtain flash redundancy or configuration information (could be in flash or OTP)
-    *   If in flash, this information is unscrambled
-    *   This item is TBD and must be further discussed.
-*   Program redundancy or configuration information into the flash controller and lock it from further modification (this item is TBD).
-*   Initialize flash scrambling and program any excluded regions based on OTP or other information.
+Once the boot ROM is allowed to execute, it is expected to further initialize the flash prior to using it.
+This involves 4 steps:
```

Comment:
Maybe we should say something like `the following steps`?
It says 4, but there appear to be only 2 bullet points in the list below...

Reply:
ah yes good point i removed a few but didn't update the number.
i'll switch to just steps like you suggest.